src/plugin.js

```diff
@@ -36,6 +39,11 @@ const errors = {
     headline: '360 HLS video not supported on this device',
     type: '360_NOT_SUPPORTED',
     message: "Your browser/device does not support HLS 360 video. See <a href='http://webvr.info'>http://webvr.info</a> for assistance."
+  },
+  'web-vr-omnitone-init-error': {
```

Comment:
Should this just be a warning? Will audio play even if it errors?

Reply:
I think you may have the right. I will double-check it.